docs/_deploying/heroku.md

```diff
@@ -0,0 +1,279 @@
+
+```shell
+git remote -v
+# heroku	https://git.heroku.com/sharp-rain-871.git (fetch)
+# heroku	https://git.heroku.com/sharp-rain-871.git (push)
+```
+
+**3. Set environment variables on Heroku**
+
+In the Getting Started guide, you [exported your signing secret and token](http://localhost:4000/bolt-js/tutorial/getting-started#tokens-and-installing-apps) as environment variables. Now we need to set the same environment variables in your Heroku app:
+
+```shell
+heroku config:set SLACK_SIGNING_SECRET=<your-signing-secret>
+heroku config:set SLACK_BOT_TOKEN=xoxb-<your-bot-token>
+```
+
+Now that we have prepared your local app and created a Heroku app, the next step is to deploy it!
+
+---
+
+### Deploy the app
+
+To deploy the app, we're going to push your local code to Heroku, update your Slack app's settings, and say "hello" to your Heroku app. ✨
+
+**1. Deploy the app to Heroku**
+
+When deploying an app to Heroku, you'll typically use the `git push` command. This will push your code from your local repository to your `heroku` remote repository.
+
+You can now deploy your app with the command:
+
+```shell
+git push heroku main
+```
+
+> 💡 Heroku deploys code that's pushed to the [master or main branches](https://devcenter.heroku.com/articles/git-branches). Pushing to other branches does nothing.
+
+Now that your code is deployed, we need to start a web server instance using the Heroku CLI:
+
+```shell
+heroku ps:scale web=1
+```
+
+**2. Update your Slack app's setting**
+
+In the [Getting Started guide][getting-started-guide], we used ngrok as your **Request URL** for actions and events. Now we need to use your Heroku web address.
+
+Get your Heroku web address with the following command:
+
+```shell
+heroku info
+# ...
+# Web URL: https://sharp-rain-871.herokuapp.com/
+```
+
+In our example, the web address is `https://sharp-rain-871.herokuapp.com/`.
+
+Now browse to the [Slack App page](https://api.slack.com/apps) and select your app name. Next, we'll update your **Request URL** in two locations to be your web address.
+
+> 💡 Your **Request URL** ends with `/slack/events`, such as `https://sharp-rain-871.herokuapp.com/slack/events`.
+
+First, select **Interativity & Shortcuts** from the side and update the **Request URL**:
+
+![Interactivity & Shortcuts page](../assets/interactivity-and-shortcuts-page.png "Interactivity & Shortcuts page")
+
+Second, select **Event Subscriptions** from the side and update the **Request URL**:
+
+![Event Subscriptions page](../assets/event-subscriptions-page.png "Event Subscriptions page")
+
+> 💡 Free Heroku apps sleep when inactive. 💤 If your verification fails, please try it again immediately.
+
+**3. Test your Slack app**
+
+Your app is now deployed and Slack is updated, so let's try it out!
+
+Open a Slack channel that your app has joined and say "hello" (lower-case). Just like in the [Getting Started guide](/bolt-js/tutorial/getting-started#sending-and-responding-to-actions), your app should respond back.
```

Comment:
This also crossed my mind. Originally, I didn't elaborate because the Getting Started guide didn't. However, I've now added an extra sentence suggestion that the developer double-checks their **Request URL**. Updated in 158887d